modules/fedimint-wallet/src/lib.rs

```diff
@@ -690,7 +690,10 @@ impl ServerModule for Wallet {
         dbtx: &mut DatabaseTransaction,
         output: &WalletOutput,
     ) -> Result<TransactionItemAmount, ModuleError> {
-        if !is_address_valid_for_network(&output.recipient, self.cfg.consensus.network) {
+        if !&output
```

Comment:
Why is that `&` here? function calls typically do all the referencing/dereferencing themselves.

Reply:
Ah, thanks for spotting this. I've now removed it.